lib/ansible/modules/packaging/os/yum.py

```diff
@@ -1166,16 +1170,37 @@ def ensure(module, state, pkgs, conf_file, enablerepo, disablerepo,
 
 def main():
 
-    # state=installed name=pkgspec
-    # state=removed name=pkgspec
-    # state=latest name=pkgspec
-    #
-    # informational commands:
-    #   list=installed
-    #   list=updates
-    #   list=available
-    #   list=repos
-    #   list=pkgspec
+
+    # The coments bellow are legacy comments.
+    """
+     state=installed name=pkgspec
+     state=removed name=pkgspec
+     state=latest name=pkgspec
+
+      The problem with list option of yum module right now is non standard approach for an ansible module.
```

Comment:
@abadger  do you know in general who is the best person to consult with before taking any decisions attempts? Would you be available for a quick voice/video chat e.g. bluejeans some day during next week maybe.